### PR TITLE
adding interface to limit mach

### PR DESCRIPTION
**Pull request by inigolcanalejo**
An interface to the limit mach number is added

### CODE REVIEW

File: applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp [hunk 1]
```diff
@@ -95,6 +95,9 @@ void KratosCompressiblePotentialFlowApplication::Register()
   KRATOS_REGISTER_3D_VARIABLE_WITH_COMPONENTS(WAKE_NORMAL);
   KRATOS_REGISTER_3D_VARIABLE_WITH_COMPONENTS(WING_SPAN_DIRECTION);
 
+  // Solver parameters
+  KRATOS_REGISTER_VARIABLE(LIMIT_MACH)
```
Comment (philbucher) [inline applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp]:
```suggestion
  KRATOS_REGISTER_VARIABLE(LIMIT_MACH_NUMBER)
```
IMO `LIMIT_MACH` sounds like a boolean

Comment (inigolcanalejo) [reply applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp]: LIMIT_MACH is a double taking the value of a prescribed limit mach number. It was hard coded to 0.94 and here it is made user defined.

Comment (marcnunezc) [reply applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp]: I think that @philbucher is referring to the fact that the variable starts with a verb, making it look like a boolean, like if you did:
`bool limit_mach = false/true;`
As you would for instance:
`bool is_wake = false/true;`


However, I think @inigolcanalejo  was trying to be consistent with the already defined variable at the fluid app that is defined as `MACH` only:
https://github.com/KratosMultiphysics/Kratos/blob/3e5b2c9eb7fe991471adb81c8f13f28d31e9d539/applications/FluidDynamicsApplication/fluid_dynamics_application_variables.cpp#L73

Comment (inigolcanalejo) [reply applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp]: I see, thanks for the clarification @marcnunezc, now I know what @philbucher meant. We could also use CLAMPED_MACH but I still find LIMIT_MACH more descriptive for what the variable does. I am open to suggestions though.

Comment (rubenzorrilla) [reply applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp]: `MACH_LIMIT` ?

Comment (inigolcanalejo) [reply applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp]: Good idea, I didn't think about that one :sweat_smile:, thanks @rubenzorrilla!

**Review by marcnunezc — APPROVED** [review]
Ok to me!

**Pull request merged**